src/main/java/io/cdap/plugin/db/batch/action/vertica/export/VerticaBulkExportAction.java

```diff
@@ -104,16 +118,18 @@ public void run(ActionContext context) throws Exception {
   }
 
   /**
-   * Vertica config
+   * Vertica Export config
    */
-  public class VerticaConfig extends PluginConfig {
+  public static class VerticaExportConfig extends PluginConfig {
```

Comment:
1. move this class to upper level (make not inner)
2. class should be immutable
3. add builder